examples/energy-management-app/linux/README.md

```diff
@@ -595,8 +595,9 @@ Step-by-step:
 
 1. Set the default TestEventTrigger (`0x0094000000000000`):
 
--   `0x0094000000000000` corresponds to `kBasicInstallationTestEvent` from
-    `WaterHeadermanagementTestEventTriggerHandler.h`
+-   `0x0094000000000000` corresponds to
+    [`kBasicInstallationTestEvent`](https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/water-heater-management-server/WaterHeaterManagementTestEventTriggerHandler.h#L47)
```

Comment:
nit: The `https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/water-heater-management-server/WaterHeaterManagementTestEventTriggerHandler.h#L47` line might change with time, so it's better to use line link reference with a commit sha instead of master.